packages/frontend/components/atomic/Dropdown/Dropdown.tsx

```diff
@@ -8,18 +8,24 @@ type BaseProps = React.ComponentProps<typeof BaseDropdown>;
  * A dropdown for navigation submenus.
  */
 const Dropdown = ({ disclosure, menuItems, label }: Props) => {
-  const submenu = menuItems.map((item, i) => {
-    if (item === null) return null;
-    return (
-      <Styled.Item key={i}>{item && React.cloneElement(item)}</Styled.Item>
-    );
-  });
+  const submenu = (onClick: React.MouseEventHandler | undefined) => {
+    return menuItems.map((item, i) => {
+      if (item === null) return null;
+      return (
+        <Styled.Item key={i}>
+          {item && React.cloneElement(item, { onClick })}
```

Comment:
There was a small problem here where a child's click event handler was getting overwritten. So if you had a button in the menu, any on click event would be changed to the hide function. I moved the onClick to the dropdown wrapper 👍